apps/ledger-live-desktop/src/renderer/analytics/segment.js

```diff
@@ -76,8 +77,7 @@ function getAnalytics() {
 }
 
 export const start = async (store: *) => {
-  if (!user || (!process.env.SEGMENT_TEST && (process.env.MOCK || process.env.PLAYWRIGHT_RUN)))
-    return;
+  if (!user || (!process.env.SEGMENT_TEST && (getEnv("MOCK") || getEnv("PLAYWRIGHT_RUN")))) return;
```

Comment:
Same question for `process.env.SEGMENT_TEST`?

Reply:
Didn't touch it because `SEGMENT_TEST` isn't part of the `env.ts` file which is then feeding the `process.env` object.
And since this issue is coming from adding stuff in `env.ts` that weren't in it in the first place, I don't want to create more side effects in this PR that needs to be merged ASAP. 
But except for `NODE_ENV`, I believe we need to have a discussion about centralizing all our environment variables, and all of them should go through a `getEnv`